src/settings/Api.ts

```diff
@@ -43,10 +43,68 @@ export class Api {
 		return $.getJSON(this.getUrl('folders'))
 			.then((data: OCSResult<Folder[]>) => Object.keys(data.ocs.data).map(id => data.ocs.data[id]));
 	}
-
+	// Returns all NC groups
 	listGroups(): Thenable<Group[]> {
 		return $.getJSON(this.getUrl('delegation/groups'))
-			.then((data: OCSResult<Group[]>) => data.ocs.data);
+			.then((data: OCSResult<Group[]>) => {
+				// No need to present the admin group as it is automaticaly added
+				const groups = data.ocs.data.filter(g => g.id !== 'admin')
+				return groups
+			});
+	}
+
+	// Returns all groups that have been granted delegated admin rights on groupfolders
+	listDelegatedAdmins(): Thenable<Group[]> {
+		return $.getJSON(this.getUrl('/delegation/authorized-groups'))
```

Comment:
for new code instead of using `$.getJson`, it's nicer to use https://github.com/nextcloud/nextcloud-axios/ (optional)

Reply:
Yes, I can use axios, I already use it for the Workspace app. I didn't use it here because I see that you use JQuery for requests ?

Maybe are you planning to migrate on nextcloud/axios ?

Reply:
I fixed it in this commit : https://github.com/nextcloud/groupfolders/pull/2072/commits/fead9d5a860c717606556e3b7b25b670ae0e64b4